selfservice/strategy/code/strategy_login.go

```diff
@@ -494,7 +494,7 @@ func (s *Strategy) loginVerifyCode(ctx context.Context, f *login.Flow, p *update
 	if err := s.verifyAddress(ctx, i, Address{
 		To:  loginCode.Address,
 		Via: loginCode.AddressType,
-	}); err != nil {
+	}, true); err != nil {
```

Comment:
In the login flow, we need to persist because the user is otherwise not persisted in the database.